src/System.Collections/ref/System.Collections.cs

```diff
@@ -107,8 +107,6 @@ public partial class Dictionary<TKey, TValue> : System.Collections.Generic.IColl
         void System.Collections.IDictionary.Remove(object key) { }
         System.Collections.IEnumerator System.Collections.IEnumerable.GetEnumerator() { throw null; }
         public bool TryGetValue(TKey key, out TValue value) { throw null; }
-        public TValue GetValueOrDefault(TKey key) { throw null; }
-        public TValue GetValueOrDefault(TKey key, TValue defaultValue) { throw null; }
```

Comment:
> GetValueOrDefault is an extension Method for IDictionary that is why we are removing the instance methods, so the tests would still pass

Right, but I'm wondering if we now have duplicate tests, i.e. did we previously have tests before for Dictionary.GetValueOrDefault and for the IDictionary extension, and now they're duplicative and just testing the latter, such that we could remove half of them?